packages/test/jest.test.js

```diff
@@ -27,36 +27,96 @@ function compareTimestamps(ts1, ts2) {
   return expect(Math.abs(ts1 - ts2)).toBeLessThan(1e2);
 }
 
-let entries, id1, id2, id3;
+let websocket, entries, id1, id2, id3, waitMessage;
+
+beforeAll(async () => {
+  websocket = new WebSocket("ws://localhost:4000");
+
+  let resolveWaitMessage = Function.prototype;
+  waitMessage = () =>
+    new Promise(resolve => {
+      resolveWaitMessage = resolve;
+    });
+
+  websocket.onmessage = event => {
+    resolveWaitMessage(jrp.parse(event.data));
+  };
+
+  await new Promise(resolve => {
+    websocket.onopen = resolve;
+  });
+});
+
+afterAll(() => {
+  websocket.close();
+});
 
 test("create entry", async () => {
-  id1 = await call("createEntry", { name: "name1", content: "content1" });
-  id2 = await call("createEntry", { content: "content2" });
-  id3 = await call("createEntry", { name: "name3" });
+  let message1;
+  [message1, id1] = await Promise.all([
+    waitMessage(),
+    call("createEntry", { name: "name1", content: "content1" }),
+  ]);
+  expect(message1.type).toBe("notification");
+  expect(message1).toEqual({
+    jsonrpc: expect.any(String),
```

Comment:
Why `any(String)`?

Reply:
I thought the version of json-rpc protocol could change, but it MUST stay the same ('2.0'). I change this!